Common/Sources/View/Error/ErrorView.swift

```diff
@@ -71,6 +86,16 @@ public final class ErrorView {
     }
 
     private func connectionError(_ handler: @escaping (UIAlertAction) -> Void) {
+        if !pageName.isEmpty, !pageGroup.isEmpty {
+            let errorInfo = ErrorInfo(type: errorTypeConnection,
+                                      message: self.error.errorDescription ?? "",
```

Comment:
Should be `error.getHyperwalletErrors()?.errorList?.first?.message`

Reply:
fixed